src/main/resources/migrations/00037_alter_table_events_add_external_id.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:alter_table_events_add_external_id
+ALTER TABLE events ADD COLUMN IF NOT EXISTS external_id VARCHAR(26);
```

Comment:
Needs upgrading PostgreSQL which supports this syntax

Reply:
Currently we use `govukpay/postgres:9.4.4` in https://github.com/alphagov/pay-direct-debit-connector/blob/ec030adff4fa88fd027c70ca5b7d92e70799d995/src/test/java/uk/gov/pay/directdebit/junit/DropwizardAppWithPostgresRule.java#L30, but `IF NOT EXISTS` was added in `9.6` https://www.postgresql.org/docs/9.6/static/sql-altertable.html

We need to rebuild the image with the new version and push it to DockerHub or use 9.4.4 syntax